plugin/src/_rules/position.js

```diff
@@ -119,3 +123,33 @@ export const insets = [
     [/^inset-([xy])-(.+)$/, handleInsetValues],
     [/^(top|left|right|bottom)-(.+)$/, ([, d, v], ctx) => ({ [d]: handleInsetValue(v, ctx) })],
 ];
+
+export const floats = [
+  // floats
+  ['float-left', { float: 'left' }],
+  ['float-right', { float: 'right' }],
+  ['float-none', { float: 'none' }],
+  ...makeGlobalStaticRules('float'),
+  // clears
+  ['clear-left', { clear: 'left' }],
+  ['clear-right', { clear: 'right' }],
+  ['clear-both', { clear: 'both' }],
+  ['clear-none', { clear: 'none' }],
+  ...makeGlobalStaticRules('clear'),
+];
+
+function handleZIndexValue(v, { theme }) {
+  if (!theme.zIndex?.[v]) return warnOnce(`${v} is not allowed as z-index value`)
+  return theme.zIndex[v]
+}
+
+export const zIndexes = [
+  [/^z-(\d+)$/, ([, v], ctx) => ({ 'z-index': handleZIndexValue(v, ctx) }), { autocomplete: ['z-$zIndex', 'z-<num>'] }],
```

Comment:
I don't think `z-<num>` should be in autocomplete? `z-$zIndex` is already covering what we allow for values.

Reply:
Ah, good point! Removed `z-<num>` in [7aea540](https://github.com/warp-ds/engine/pull/10/commits/7aea5404f614ef99939988899568bd933119453b)